docs/ecosystem/roadmap.md

```diff
@@ -0,0 +1,38 @@
+---
+id: roadmap
+title: Roadmap
+---
+
+## ORY Roadmap
+
+If you would like a quick overview of whats coming next to ORY, you can do so at our github project page.  
+You can see what we are working on right now, what issues are prioritized and what progress has been made in the past.  
+[Follow this link for the complete Project Roadmap](https://github.com/orgs/ory/projects/8).
+
+## Complete Roadmap Sections: 
+
+[To Do](https://github.com/orgs/ory/projects/8#column-6092768)  
+[Priority](https://github.com/orgs/ory/projects/8#column-7739379)  
+[In Progress](https://github.com/orgs/ory/projects/8#column-6092769)  
+[Review In Progress](https://github.com/orgs/ory/projects/8#column-6092770)  
+[Reviewer Approved](https://github.com/orgs/ory/projects/8#column-6092771)  
+[Done](https://github.com/orgs/ory/projects/8#column-6092772)  
+
+## Individual Project Sections
+
+[ORY Kratos Milestones](https://github.com/ory/kratos/milestones)  
+[ORY Kratos Pull Requests](https://github.com/ory/kratos/pulls)  
```

Comment:
You can combine PR and issues using filters like so: https://github.com/ory/kratos/issues?q=is%3Aopen

You can also sort by recent and updated recently: is:open sort:updated-desc sort:reactions-+1-desc

Reply:
Ok, i sorted by "is open" now, it seems to be the most straightforward option.